api/handler/service.go

```diff
@@ -409,11 +410,17 @@ func (s *ServiceAction) ServiceHorizontal(hs *model.HorizontalScalingTaskBody) e
 		logrus.Errorf("get service by id %s error, %s", service.ServiceID, err)
 		return err
 	}
+
 	// for rollback database
 	oldReplicas := service.Replicas
-	if int32(service.Replicas) == hs.Replicas {
-		return nil
+	pods, err := s.statusCli.GetServicePods(service.ServiceID)
+	if err != nil {
+		return pkgerr.Wrap(err, "GetPodByService Error")
 	}
+	if int32(len(pods.NewPods)) == hs.Replicas{
+		return fmt.Errorf("horizontal service faliure: no change, no update")
```

Comment:
only new pods? should return an error?

Reply:
Yes, a new pod refers to a pod that is running or updated. When it is not being updated, the new pod is the current number of instances. An error must be returned, otherwise the page will be displayed as horizontal scaling in progress, and there is no log